src/main/java/model/Status.java

```diff
@@ -0,0 +1,11 @@
+package model;
+
+public enum Status {
```

Comment:
If it's a separate class file then better use more specific name.
```suggestion
public enum OrderStatus {
```